### ISSUE TITLE
Visual Accessibility issues on Model properties

### ISSUE THREAD

**Issue by maryscar** [swagger-api/swagger-ui#4201]
| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  Bug
| Which Swagger/OpenAPI version?  | Latest
| Which Swagger-UI version?       | 3.9.3
| How did you install Swagger-UI? | swagger-net
| Which browser & version?        | Chrome 64.0.3282.140
| Which operating system?         | Windows 10 v1709

### Expected Behavior
1 ) Class property-format span fails color contrast ratio thresholds for accessibility.
2 ) Property Style fails color contrast ratio thresholds for accessibility.

### Current Behavior
1 ) 
![image](https://user-images.githubusercontent.com/34582625/35883382-ee1368c4-0b3b-11e8-9abf-7ac4006fc84d.png)

2 ) 
![image](https://user-images.githubusercontent.com/34582625/35883397-fcb2cfb4-0b3b-11e8-84f5-d38e55e9cac8.png)

### Possible Solution
1 ) Increase the weight to the property format. 
$prop-format-font-color: $gray-600 !default;
2 ) Increase the weight to the property format. 
From #999 to #6b6b6b

### Context
Accessibility testing was completed using internal tools as well as Google Accessibility Extension (https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb) and aXe Accessibility Engine (https://www.deque.com/products/axe/). These failures were the most consistent and effect users with visual accessibility issues. Will submit a pull request with fixes.

**Comment by maryscar** [comment]
PR for fixes merged.
